auth/client/src/test/java/org/wildfly/security/auth/client/DefaultSSLContextProviderDoesNotLoopTestCase.java

```diff
@@ -0,0 +1,33 @@
+package org.wildfly.security.auth.client;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.wildfly.client.config.ConfigXMLParseException;
+
+import javax.net.ssl.SSLContext;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.security.GeneralSecurityException;
+import java.security.NoSuchAlgorithmException;
+import java.security.Security;
+
+public class DefaultSSLContextProviderDoesNotLoopTestCase {
+    private static final String CONFIG_FILE = "file:./src/test/resources/org/wildfly/security/auth/client/test-wildfly-config-default-ssl-context-invalid-looping.xml";
+
+    @Test
+    public void testDefaultSSLContextProviderDoesNotLoopTestCase() throws GeneralSecurityException, URISyntaxException, ConfigXMLParseException {
+        Security.insertProviderAt(new ElytronClientDefaultSSLContextProvider(), 1);
```

Comment:
Isn't there the need to check that the registration went well as it is done in the previous test cases?

Same question about `DefaultSSLContextProviderFromCurrentAuthContextTestCase`.

Reply:
@fabiobrz Added, thanks.